bears/js/ESLintBear.py

```diff
@@ -48,7 +48,7 @@ def generate_config(filename, file):
         return '{"extends": "eslint:recommended"}'
 
     def process_output(self, output, filename, file):
-        if not file:
+        if not file or not output:
             return
```

Comment:
I know, but I do not have stderr here. SO, the message is empty ...
I will be solving https://github.com/coala-analyzer/coala/issues/2711 and https://github.com/coala-analyzer/coala-bears/issues/730 soon after. But this JSONDecodeError is another issue which Im fixing here